### PR TITLE
fixes for github commit fetching

### PR DESCRIPTION
**Pull request by huerlisi**

### CODE REVIEW

**Comment by huerlisi** [comment]
Ready for review.

File: jobs/github.rb [hunk 1]
```diff
@@ -7,8 +7,9 @@ def update
 
   # get all today's events
   last_response = client.last_response
-  while events.last.created_at.to_date == Date.today
-    last_response = last_response.rels[:next].get
+  while events.last.created_at.to_date >= Date.today
+    break unless next_page = last_response.rels[:next]
```
Comment (koffeinfrei) [inline jobs/github.rb]: I'd rather like `while events.last.created_at.to_date >= Date.today && last_response.rels[:next]`

**Comment by huerlisi** [comment]
@koffeinfrei fixed

**Comment by koffeinfrei** [comment]
:+1: I refactored the github job into a proper class, will manually integrate and push after you're merged.

**Pull request merged**